### PR TITLE
[DNM] Overmap Autopilot Adjustments

### PR DESCRIPTION
**Pull request by bloxgate**
Port Baystation12#23846

### CODE REVIEW

**Comment by BDpuffy420** [comment]
Conflict in helm.dm

**Comment by bloxgate** [comment]
Marking this DNM until we're sure if #1560 was successful or not.

File: code/modules/overmap/ships/ship.dm [hunk 1]
```diff
@@ -90,7 +90,7 @@
 	return get_speed()/get_acceleration()
 
 /obj/effect/overmap/ship/proc/decelerate()
-	if(!is_still() && can_burn())
+	if(((speed[1]) || (speed[2])) && can_burn())
```
Comment (caelaislinn) [inline code/modules/overmap/ships/ship.dm]: What's the purpose of removing the call to the helper? This is identical code anyway

**Review by caelaislinn — CHANGES_REQUESTED** [review]

File: code/modules/overmap/ships/ship.dm (hunk 1 above)
Comment (bloxgate) [reply code/modules/overmap/ships/ship.dm]: Ah, that seems to be something that was done in the PR on Bay. Not sure really.

The only changes I've made were to get it to merge with our helm computer, as it was slightly different.

**Comment by BDpuffy420** [comment]
#1560  has partly worked but is still an issue, would recommend merging this

**Review by caelaislinn — APPROVED** [review]

**Pull request merged**